integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseBuyDashFragment.kt

```diff
@@ -126,17 +122,12 @@ class CoinbaseBuyDashFragment : Fragment(R.layout.fragment_coinbase_buy_dash) {
             safeNavigate(CoinbaseServicesFragmentDirections.coinbaseServicesToError(placeBuyOrderError))
         }
 
-        amountViewModel.dashToFiatDirection.observe(viewLifecycleOwner){ dashToFiat ->
-            analyticsService.logEvent(if (dashToFiat) AnalyticsConstants.Coinbase.ENTER_AMOUNT_DASH
-            else AnalyticsConstants.Coinbase.ENTER_AMOUNT_FIAT,
```

Comment:
This needs to be logged on the Continue click and not when the direction is switched